docs/markdown/Reference-manual.md

```diff
@@ -228,6 +228,7 @@ the `@variable@` syntax.
 - `input` the input file name. If it's not specified in configuration
   mode, all the variables in the `configuration:` object (see above)
   are written to the `output:` file.
+- `install` when true, this generated file is installed during the install step
```

Comment:
The behavior changed here, please add something like `*changed in (0.49.0)* install now has precedence over install_dir`

Reply:
Sorry for nitpicking, but I think we should consider `install` kwarg to be a new addition in 0.49.0, because if was not documented and not working before.

So what about something like:
`install` *(added 0.49.0)* When true, this generated file is installed during the install step, and `install_dir` must be set and not empty. When false, this generated file is not installed regardless of the value of `install_dir`. When omitted it defaults to true when `install_dir` is set and not empty, false otherwise.